src/Utils/utils.ts

```diff
@@ -561,3 +562,26 @@ export function omitBy<T extends Record<string, unknown>>(
     Object.entries(obj).filter(([_, value]) => !predicate(value)),
   ) as Partial<T>;
 }
+
+export const copyToClipboard = async (
+  text: string,
+  successMsg: string,
+  errorMsg: string,
+  onSuccess: () => void,
+  onResetIcon: () => void,
+  resetIconDuration = 2500,
+) => {
```

Comment:
make the function do only what it's meant to do.

```jsx
export const copyToClipboard = async (content: string) => { 
  try {
    await navigator.clipboard.writeText(text);
  } catch (err) {
    Notification.Error({ msg: "Copying is not allowed" });
  }
}
```

Reply:
sorry for the clutter @rithviknishad, I have made the required changes as requested